components/button/button.css

```diff
@@ -0,0 +1,78 @@
+.button, 
+.button.large,
+.button.button-solid {
+  border-radius: 5px;
+  border: none;
+  background-color: rgb(6, 140, 109);
+  color: white;
+  padding: 0.8rem 1.25rem;
+  font-weight: 500;
+  font-size: 1.25rem;
+}
+
+.button:hover {
+  background-color: rgba(6, 140, 109, 0.8);
+}
+
+.button.small {
+  padding: 0.625rem;
+  font-weight: 500;
+  font-size: 0.8rem;
+}
+
+.button.mid {
+  padding: 0.8rem;
```

Comment:
Can use pixels for small size instead of rems.

Reply:
will do the required refactoring. Thanks:)